multiphate/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: multiphate 
+  version: 0.5 
+
+source:
+  git_url: https://github.com/carolzhou/multiPhATEb.git
```

Comment:
Can you create a release and use a release tarball instead?

Thanks!

Reply:
Yes, I have created a release tarball and resubmitted the recipe for building.

Reply:
Actually, I though the build failed because of the git_url. ???

Reply:
And I thought another PR failed due to an emboss problem.  ??? I have to say this is a confusing process.  (Sorry to complain.)

Reply:
? I don't see any changes here also no emboss failure.

Reply:
This is the error message I copied out, and why I re-submitted a PR. Maybe I should let the most recent version go through again, since I changed the meta.yaml (with a copy in the multiphate.tar.gz), which changes the sha256.  Sorry for the confusion.  My first time with Conda/bioconda packaging.  07:01:40 BIOCONDA ERROR BUILD SUMMARY: SKIPPED recipe multiphate due to failed dependencies ['recipes/perl-file-path', 'recipes/perl-file-temp/0.2304', 'recipes/perl-test-simple', ‘recipes/emboss']

Reply:
That was for #13872

Reply:
Why haven't you used this URL instead? https://github.com/carolzhou/multiPhATEb/archive/a706cac23339588fe9fa4a34d607fcdcabccfe5a.zip